README.md

```diff
@@ -65,6 +65,11 @@ brew install grype
 
 **Note**: Currently, Grype is built only for macOS and Linux.
 
+### On Github Actions
+
+`grype` has been packaged as a Github Action within [Anchore Container Scan](https://github.com/marketplace/actions/anchore-container-scan)
```

Comment:
We typically refer to this as the "Scan Action", as it does more than just containers. And a more pertinent link might just be to the repository? https://github.com/anchore/scan-action

Reply:
This one is a little nuanced, since we're shown in the GitHub Actions marketplace as "Anchore Container Scan" (even though the repo itself is "scan-action"). This may be worth an offline conversation about updating our marketplace presence.